src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,63 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  max-width: 706px;
+  padding: 10px;
+  gap: 1px;
+
+  @each $day, $margin in $start-day {
```

Comment:
The variable `$start-day` is used here, but it is not defined within this file. Ensure that it is defined elsewhere or imported correctly to avoid compilation errors.